src/stats/cdf.rs

```diff
@@ -0,0 +1,211 @@
+// Copyright 2016 Johannes Köster.
+// Licensed under the MIT license (http://opensource.org/licenses/MIT)
+// This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Support for discrete probability distributions in terms of cumulative distribution
+//! functions (CDF).
+
+use std::f64;
+use std::iter;
+use std::slice;
+
+use num::traits::{cast, NumCast};
+use itertools::Itertools;
```

Comment:
This one yes, but the import in the test module was superfluous. I removed it. Thanks!